package.json

```
@@ -82,6 +82,11 @@
                         "default": true,
                         "description": "Controls whether the editor should show diagnostics, such as warnings for unknown patterns."
                     },
+                    "hex-casting.showInternalNameHints": {
+                        "type": "boolean",
+                        "default": false,
+                        "description": "Controls whether the editor should display the internal name of each pattern."
```

```suggestion
                        "description": "Enable/disable inlay hints for internal pattern names:\n```hexpattern\nMind's Reflection/* get_caster*/\n```"
```